src/js/components/DataSearch/stories/Simple.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { Data, DataSummary, DataTable, Grid, Paragraph } from 'grommet';
+
+import { DataSearch } from '../DataSearch';
+import { columns, DATA } from '../../DataTable/stories/data';
+
+export const Simple = () => (
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Grid pad="large" columns={['large']} justifyContent="center">
+    <Paragraph color="text-weak">
+      Note: Results are filtered as you type, checking all fields.
+    </Paragraph>
+    <Data data={DATA} updateOn="change">
+      <DataSearch />
```

Comment:
```suggestion
      <DataSearch aria-label="Search to filter DataTable" />
```
Right now voice over is just reading this as 'Search text field'. For a screenreader user I think it would bee unclear what this field is searching or that it is associated with the DataTable that is farther down the page. In our stories it might be nice to include an aria-label to showcase the practice of including an aria-label so that a screenreader user has more understanding of what this field is searching.

Reply:
What about giving a `name` or `label` to `<Data />` to help us build better accessible text in sub components?

Reply:
So you would pass name or label to `<Data>` and then whatever you pass will be used in the `aria-label` for `<DataSearch>` and possibly other components?

Reply:
Added `messages` property, also added default messages in default.json.